package.json

```diff
@@ -3,29 +3,31 @@
   "version": "1.0.0",
   "main": "gulpfile.js",
   "config": {
-    "hugo": {
-      "en": "config.toml",
-      "es": "config_es.toml"
-    },
-    "urls": {
-      "en": {
-        "staging": "https://vote-gov-staging.apps.cloud.gov/",
-        "production": "https://vote-gov.apps.cloud.gov/"
+    "votgov": {
```

Comment:
I think you have a typo here - should be `votegov` (which you've used in the vars below), not `votgov`